lib/rails/observers/active_model/observing.rb

```diff
@@ -14,8 +14,13 @@ module Observing
 
     included do
       extend ActiveSupport::DescendantsTracker
+      def self.inherited(subclass)
+        super
+        ActiveRecord::Base.instantiate_observers
```

Comment:
Since this is in ActiveModel, not activerecord; should it be self.instantiate_observers?

Also in the railtie; there's a lot of loading and explicitly instantiate_observing going on; but nothing that specifically seems to target active_model itself